pydash/functions.py

```diff
@@ -10,7 +10,7 @@
 import time
 
 import pydash as pyd
-from ._compat import _range
+from ._compat import _range, PY3
```

Comment:
Instead of doing the `PY3` check in each module, I'd instead define `getfullargspec` in `_compat.py` and import that function where needed:

```python
# _compat.py
if PY3:
    getfullargspec = inspect.getfullargspec
else:
    getfullargspec = inspect.getargspec
```